util/config/gather-clang-sysroot-arguments

```diff
@@ -28,4 +28,12 @@ if [ $? -eq 0 ]; then
   fi
 fi
 
+TARGET_PLATFORM=`$CHPL_HOME/util/chplenv/chpl_platform.py --target`
+if [[ $TARGET_PLATFORM -eq "darwin" ]]; then
```

Comment:
I think maybe you meant to use `==` here? I'm seeing errors on a SLES 12 system about `sw_vers` not being found.

Reply:
OK, thanks for reporting.  I think you're probably right (though I thought I'd just copied this off the internet somewhere and had to google `eq` vs. `==` to know what the difference is myself).  Just out of curiosity, what does `$CHPL_HOME/util/chplenv/chpl_platform.py --target` return on that system?

Reply:
```
$ ./util/chplenv/chpl_platform.py --target
linux64
```

Reply:
That's what I was guessing...  I wonder why we didn't have far more explosions in testing...  Is the definition of `-eq` on non-numeric strings undefined?  Or maybe it changed somehow uniquely in SLES 12?

In any case, I agree that `==` looks like what I should've used, and opened #17876 to address.